src/components/Cards/BountyCard.tsx

```diff
@@ -97,6 +98,8 @@ export function BountyCard({ data }: Props) {
       ? expiredBadge
       : countdownBadge;
 
+  const videoMetadata = image.type === 'video' ? (image.metadata as VideoMetadata | null) : null;
```

Comment:
`(image.metadata ?? {}) as VideoMetadata` personal preference but I think this works / looks better. That way you can at least expect it to be defined.